.github/workflows/build-fd-image.yaml

```diff
@@ -6,7 +6,7 @@ on:
       docker_tag_version:
         description: 'Fluentd image release version'
         required: true
-        default: '1.15.3'
+        default: '1.17.0'
```

Comment:
https://github.com/fluent/fluent-operator/blob/master/.github/workflows/build-fb-image.yaml#L9
fluentbit's image need to be updated too